src/gui/guiScrollBar.cpp

```diff
@@ -183,36 +183,41 @@ void GUIScrollBar::draw()
 			slider_rect.LowerRightCorner.Y =
 					slider_rect.UpperLeftCorner.Y + thumb_size;
 		}
-		skin->draw3DButtonPaneStandard(this, slider_rect, &AbsoluteClippingRect);
+		
+		// Clip scrollbar so it doesn't show behind buttons for elastic scrolling
```

Comment:
I wonder if it would look better if the slider was not clipped but squished. :thinking: 
(Not really relevant right now, but maybe at some point when there's scrollbar styling.)

Reply:
You're right. In fact, I would consider this relevant. If it was squished that would probably make it look more natural where as the clip seems a bit lazy.
When I get back to this, I'll probably actually do that.